analysis/measure_opioid_prescription_martina.py

```diff
@@ -0,0 +1,146 @@
+# This script will
+# To determine the rate of opioid prescribing in England, January 2019 to December 2022 by age group
+# How many PEOPLE per 100,000 population are prescribed an opioid per month?
+# How many OPIOIDS are prescribed per 100,000 population per month?
+# How many PEOPLE per 100,000 population initiate a NEW OPIOID treatment episode per month?
+# How do the above measures vary by AGE group (10-year intervals) and sex?
+
+from ehrql import (
+    case,
+    INTERVAL,
+    create_measures,
+    months,
+    when,
+    codelist_from_csv,)
+
+
+from ehrql.tables.tpp import (
+    medications,
+    patients,
+    practice_registrations,
+    )
+
+
+# Every measure definitions file must include this line
+measures = create_measures()
+
+#index date
+index_date = INTERVAL.start_date
+
+
+# Disable disclosure control for demonstration purposes.
+# Values will neither be suppressed nor rounded.
+measures.configure_disclosure_control(enabled=False)
+
+
+#Import codelist
+opioid_analgesia_codelist = codelist_from_csv(
+    "codelists/user-anschaf-opioids-for-analgesia-dmd.csv",
+    column="code",
+)
+
+
+# Variables
+## Sex
+has_recorded_sex = patients.sex.is_in(["male", "female"])
+
+
+## Age
+age = patients.age_on(index_date)
+
+
+age_band = case(
+    when((age >= 0) & (age < 20)).then("0-19"),
+    when((age >= 20) & (age < 40)).then("20-39"),
+    when((age >= 40) & (age < 60)).then("40-59"),
+    when((age >= 60) & (age < 80)).then("60-79"),
+    when(age >= 80).then("80+"),
+)
+
+
+# Denominator: For all objectives, the denominator is
+# adults (18-110 years)
+was_adult = (patients.age_on(index_date) >= 18) & (
```

Comment:
```suggestion
was_adult = (age >= 18) & (
```
As you created the "age" variable above, no need to use "patients.age_on(index_date)"